package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@tacc/core-cms-resources",
+  "version": "3.14.0",
+  "license": "MIT",
+  "description": "Any custom code for all new and updated TACC CMS sites.",
+  "scripts": {
+    "build": "npm run build:css --project=$npm_config_project",
+    "build:css": "_shared/bin/build-css-via-cli.js",
+    "update-version": "_shared/bin/update-version.js"
+  },
+  "// scripts": {
```

Comment:
Not really. Prepending an otherwise duplicate key string with `// ` is my own practice to allow me to add comments about content of an object in JSON (which does not natively support comments). Check within to see that I have described each script. These comments are likely superfluous to Node-familiar devs. Drop 'em?